config/crd/kustomization.yaml

```diff
@@ -16,15 +16,15 @@ patchesStrategicMerge:
 #- patches/webhook_in_csiaddonsnodes.yaml
 #- patches/webhook_in_reclaimspacejobs.yaml
 #- patches/webhook_in_volumereplications.yaml
-- patches/webhook_in_volumereplicationclasses.yaml
+#- patches/webhook_in_volumereplicationclasses.yaml
 #+kubebuilder:scaffold:crdkustomizewebhookpatch
 
 # [CERTMANAGER] To enable cert-manager, uncomment all the sections with [CERTMANAGER] prefix.
 # patches here are for enabling the CA injection for each CRD
 #- patches/cainjection_in_csiaddonsnodes.yaml
 #- patches/cainjection_in_reclaimspacejobs.yaml
 #- patches/cainjection_in_volumereplications.yaml
-- patches/cainjection_in_volumereplicationclasses.yaml
+#- patches/cainjection_in_volumereplicationclasses.yaml
```

Comment:
This should be uncommitted for enabling webhook conversion, 
since we don't have conversion for any CRDs, we don't need this.